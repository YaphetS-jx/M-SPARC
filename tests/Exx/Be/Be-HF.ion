ATOM_TYPE: Be
PSEUDO_POT: Be.pot
N_TYPE_ATOM: 1
COORD:
10 10 10
