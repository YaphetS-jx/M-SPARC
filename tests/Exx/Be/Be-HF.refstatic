***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
     10.0000000000      10.0000000000      10.0000000000
Atomic forces (Ha/Bohr):
      0.0000000000       0.0000000000       0.0000000000
