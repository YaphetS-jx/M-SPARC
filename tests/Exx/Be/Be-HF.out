***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-Jan-2021  Start time: 16:34:40                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 20.000000 20.000000 
FD_GRID: 67 67 67
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999993
CHEB_DEGREE: 26
NSTATES: 3
EXCHANGE_CORRELATION: HF
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
MAXIT_FOCK: 20
TOL_FOCK: 1.00E-06
HYB_MIXING: 1.00
EXX_METHOD: FOURIER_SPACE
OUTPUT_FILE: ../tests/Exx/Be/Be-HF.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.298507 (Bohr)
Output printed to                  :  ../tests/Exx/Be/Be-HF.out
Total number of atom types         :  1
Total number of atoms              :  1
Total number of electrons          :  4
Atom type 1  (valence electrons)   :  Be 4
Pseudopotential                    :  Be.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.87 6.87 6.87
Number of atoms of type 1          :  1
Estimated total memory usage       :  1.18 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.3095230669E+01        3.739E-01        6.261
2            -1.3164357756E+01        1.560E-01        9.296
3            -1.3172858464E+01        1.096E-01        9.248
4            -1.3172626958E+01        8.327E-02        8.108
5            -1.3173084811E+01        3.261E-02        7.432
6            -1.3172958834E+01        3.034E-02        7.860
7            -1.3173132019E+01        1.148E-02        6.863
8            -1.3173202435E+01        5.116E-03        6.701
9            -1.3173197823E+01        4.950E-03        7.094
10           -1.3173204652E+01        1.834E-03        6.462
11           -1.3173204870E+01        4.007E-04        6.348
12           -1.3173204790E+01        8.028E-04        4.963
13           -1.3173204828E+01        6.144E-04        5.007
14           -1.3173204865E+01        3.585E-04        4.674
15           -1.3173204878E+01        1.510E-04        5.072
16           -1.3173204883E+01        1.621E-04        4.318
17           -1.3173204879E+01        1.504E-04        4.542
18           -1.3173204881E+01        1.068E-04        4.277
19           -1.3173204871E+01        2.880E-04        4.274
20           -1.3173204877E+01        1.201E-04        4.732
21           -1.3173204882E+01        8.771E-05        4.342
22           -1.3173204880E+01        1.809E-05        4.808
23           -1.3173204883E+01        9.780E-06        3.161
24           -1.3173204883E+01        3.958E-06        3.923
25           -1.3173204883E+01        8.805E-06        3.136
26           -1.3173204883E+01        1.115E-06        3.405
27           -1.3173204884E+01        3.068E-06        3.589
28           -1.3173204884E+01        2.916E-06        2.440
29           -1.3173204883E+01        1.615E-06        2.443
30           -1.3173204883E+01        6.864E-07        2.485
Total number of SCF: 30    
No.1 Exx outer loop:
1            -1.4941744318E+01        2.219E-02        39.077
2            -1.4941770620E+01        1.350E-02        15.535
3            -1.4941786532E+01        2.401E-04        15.125
4            -1.4941786664E+01        1.458E-04        14.516
5            -1.4941786664E+01        2.290E-04        13.742
6            -1.4941786673E+01        4.962E-05        13.163
7            -1.4941786664E+01        2.354E-05        12.835
8            -1.4941786674E+01        9.433E-06        12.185
9            -1.4941786687E+01        4.315E-06        11.804
10           -1.4941786678E+01        1.263E-06        11.421
11           -1.4941786662E+01        4.471E-07        11.071
Total number of SCF: 11    
 Eex = 1.83216614
 Etot = -13.10962052
No.2 Exx outer loop:
1            -1.4939468041E+01        9.735E-03        38.313
2            -1.4939472975E+01        5.923E-03        14.473
3            -1.4939475937E+01        5.558E-05        13.891
4            -1.4939475956E+01        1.156E-04        13.891
5            -1.4939475954E+01        6.616E-06        13.121
6            -1.4939475952E+01        7.926E-06        11.902
7            -1.4939475942E+01        7.335E-06        11.071
8            -1.4939475976E+01        2.595E-06        11.780
9            -1.4939475962E+01        5.486E-07        11.118
Total number of SCF: 9     
 Eex = 1.82988251
 Etot = -13.10959345
No.3 Exx outer loop:
1            -1.4937384945E+01        4.330E-03        38.098
2            -1.4937385909E+01        2.639E-03        14.164
3            -1.4937386477E+01        4.563E-05        13.334
4            -1.4937386479E+01        2.041E-05        13.493
5            -1.4937386458E+01        5.301E-06        12.613
6            -1.4937386487E+01        3.815E-06        11.300
7            -1.4937386427E+01        8.802E-07        10.881
Total number of SCF: 7     
 Eex = 1.82827109
 Etot = -13.10911534
No.4 Exx outer loop:
1            -1.4936283346E+01        1.934E-03        38.106
2            -1.4936283552E+01        1.181E-03        13.679
3            -1.4936283663E+01        5.738E-05        13.200
4            -1.4936283646E+01        1.494E-05        13.223
5            -1.4936283653E+01        5.081E-06        12.331
6            -1.4936283680E+01        1.658E-06        11.892
7            -1.4936283664E+01        9.951E-07        10.832
Total number of SCF: 7     
 Eex = 1.82745718
 Etot = -13.10882649
No.5 Exx outer loop:
1            -1.4935764626E+01        8.675E-04        37.371
2            -1.4935764670E+01        5.308E-04        13.453
3            -1.4935764686E+01        1.490E-05        12.923
4            -1.4935764684E+01        4.753E-07        13.264
Total number of SCF: 4     
 Eex = 1.82707888
 Etot = -13.10868580
No.6 Exx outer loop:
1            -1.4935528271E+01        3.885E-04        37.629
2            -1.4935528281E+01        2.380E-04        13.335
3            -1.4935528285E+01        4.024E-06        12.920
4            -1.4935528264E+01        5.702E-07        12.952
Total number of SCF: 4     
 Eex = 1.82690740
 Etot = -13.10862086
No.7 Exx outer loop:
1            -1.4935421976E+01        1.742E-04        37.833
2            -1.4935421964E+01        1.076E-04        13.377
3            -1.4935421980E+01        1.272E-06        12.303
4            -1.4935421952E+01        3.748E-07        12.462
Total number of SCF: 4     
 Eex = 1.82683035
 Etot = -13.10859160
No.8 Exx outer loop:
1            -1.4935374227E+01        7.761E-05        36.766
2            -1.4935374206E+01        4.847E-05        12.724
3            -1.4935374212E+01        1.022E-06        11.402
4            -1.4935374204E+01        3.765E-07        11.793
Total number of SCF: 4     
 Eex = 1.82679573
 Etot = -13.10857847
No.9 Exx outer loop:
1            -1.4935352752E+01        3.403E-05        36.579
2            -1.4935352726E+01        2.157E-05        12.406
3            -1.4935352738E+01        7.523E-07        11.169
Total number of SCF: 3     
 Eex = 1.82678013
 Etot = -13.10857261
No.10 Exx outer loop:
1            -1.4935343048E+01        1.463E-05        36.772
2            -1.4935343031E+01        9.279E-06        11.918
3            -1.4935343014E+01        2.979E-07        10.919
Total number of SCF: 3     
 Eex = 1.82677310
 Etot = -13.10856991
No.11 Exx outer loop:
1            -1.4935338716E+01        6.079E-06        36.481
2            -1.4935338699E+01        3.998E-06        11.318
3            -1.4935338670E+01        3.464E-07        10.696
Total number of SCF: 3     
 Eex = 1.82676997
 Etot = -13.10856870
No.12 Exx outer loop:
1            -1.4935336800E+01        2.302E-06        36.513
2            -1.4935336796E+01        1.600E-06        11.105
3            -1.4935336771E+01        2.191E-07        10.544
Total number of SCF: 3     
 Eex = 1.82676856
 Etot = -13.10856821
No.13 Exx outer loop:
1            -1.4935335903E+01        1.186E-06        36.697
2            -1.4935335900E+01        8.301E-07        10.943
3            -1.4935335888E+01        2.935E-07        10.248
Total number of SCF: 3     
 Eex = 1.82676794
 Etot = -13.10856795
No.14 Exx outer loop:
1            -1.4935335551E+01        4.018E-07        36.568
2            -1.4935335552E+01        2.812E-07        10.536
3            -1.4935335538E+01        5.509E-09        10.246
Total number of SCF: 3     
 Eex = 1.82676767
 Etot = -13.10856787
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -1.3108567872E+01 (Ha/atom)
Total free energy                  : -1.3108567872E+01 (Ha)
Band structure energy              : -9.4919034041E+00 (Ha)
Exchange correlation energy        : -0.0000000000E+00 (Ha)
Self and correction energy         : -1.2034918322E+01 (Ha)
Entropy*kb*T                       : -1.7870471952E-10 (Ha)
Fermi level                        : -2.8838337347E-01 (Ha)
Average force                      :  0.0000000000E+00 (Ha/Bohr)
Maximum force                      :  0.0000000000E+00 (Ha/Bohr)
Time for force calculation         :  1.637 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1531.675 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
